### PR TITLE
torch 2.3 inference docker

### PR DESCRIPTION
**Pull request by Uxito-Ada**
### 4. How to test?
- [ ] N/A
- [ ] Unit test: Please manually trigger the PR Validation [here](https://github.com/intel-analytics/ipex-llm-workflow/actions/workflows/llm-PR-validation.yml) by inputting the PR number (e.g., `1234`). And paste your action link here once it has been successfully finished.
- [ ] Application test
- [ ] Document test
- [ ] ...

### 5. New dependencies

- [ ] New Python dependencies
       - Dependency1 
       - Dependency2
       - ...
- [ ] New Java/Scala dependencies and their license
       - Dependency1 and license1
       - Dependency2 and license2
       - ...

### CODE REVIEW

File: docker/llm/inference/xpu/docker/torch2.1/Dockerfile
Comment (glorysdj) [inline docker/llm/inference/xpu/docker/torch2.1/Dockerfile:1]: dont need to keep torch 2.1 version

File: docker/llm/inference/xpu/docker/torch2.3/convert-model-textgen-to-classfication.py
Comment (glorysdj) [inline docker/llm/inference/xpu/docker/torch2.3/convert-model-textgen-to-classfication.py:1]: make it an example, dont need to put it in this Docker image

File: docker/llm/inference/xpu/docker/torch2.3/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+   export http_proxy=http://your_proxy_url
+   export https_proxy=http://your_proxy_url
+
+   # This will yield gpt2-medium-classification under /llm/models in the container
+   python convert-model-textgen-to-classfication.py
+   ```
+
+Benchmark GPT2-Medium's performance with IPEX-LLM engine:
+
+   ``` sbash
```
Comment (ACupofAir) [inline docker/llm/inference/xpu/docker/torch2.3/README.md:66]: sbash -> bash

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**